build/ts/app.ts

```diff
@@ -215,8 +224,12 @@ class Application {
         .empty();
 
       for (const sticker of card.stickers) {
+        const stickerClass = sticker.Class != "" ? ` sticker-${sticker.Class}` : ""
+        const classAttribute = `sticker-${sticker.Name}${stickerClass}`
         $stickers
-          .append(`<div><img src="${sticker.Image}" alt="${sticker.Title}" title="${sticker.Title}"></div>`);
+          .append(sticker.Image == "" ?
+            `<div class="${classAttribute}">${sticker.Title}</div>` :
```

Comment:
I was thinking of the alignment to the left... And it occurred to me, it's not just a list. Rather, it's a container of two columns (left and right), and the one on the right are meant to be for icons. So, in order to solve that issue, you'd need to select right column where `$stickers` is defined.

Reply:
not sure what you mean

Reply:
It's out of scope, it's not a solution :D I'm just pointing out what the issue is at stake :)